src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/AzureManagedIdentityAuthenticationProvider.cs

```diff
@@ -0,0 +1,251 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
In general for this file, I would add some logging lines so that we can see things like entering and exiting the main routines, success and failure, perhaps unexpected return values from the endpoints, etc.

Reply:
Will do that in both providers, as both seem to lack EventSource tracing logs.

Reply:
Added